themes/godotengine/partials/footer.htm

```diff
@@ -4,9 +4,12 @@
 <footer>
   <div class="container flex">
     <div id="copyright">
-      © 2007-2020 Juan Linietsky, Ariel Manzur and <a href="https://github.com/godotengine/godot/blob/master/AUTHORS.md" target="_blank">contributors</a><br>
-      Godot is a member of the <a href="https://sfconservancy.org/" target_="_blank">Software Freedom Conservancy</a><br>
-      Kindly hosted by <a href="https://tuxfamily.org" target="_blank">TuxFamily.org</a>
+      <p>
+        © 2007-2020 Juan Linietsky, Ariel Manzur and <a href="https://github.com/godotengine/godot/blob/master/AUTHORS.md" target="_blank">contributors</a><br>
+        Godot is a member of the <a href="https://sfconservancy.org/" target_="_blank">Software Freedom Conservancy</a><br>
+        Kindly hosted by <a href="https://tuxfamily.org" target="_blank">TuxFamily.org</a>.
+      </p>
+      <p><a href="/privacy-policy">Privacy Policy</a> &nbsp; <a href="/code-of-conduct">Code of Conduct</a></p>
```

Comment:
This is a quick and dirty addition to the footer. Proper styling might be wanted eventually.